README.md

```diff
@@ -46,15 +47,15 @@ None
 
 | Variable | Default |
 |----------|---------|
-| `__java_packages` | `["java-1.8.0-openjdk"]` |
+| `__java_packages` | `["java-1.8.0-openjdk", "java-1.8.0-openjdk-devel"]` |
 
-## Ubuntu-14.04
+## Ubuntu-14
```

Comment:
why are you trying to change the meaning of the sections? you are not happy with `meta/main.yml` even when it says which version is supported. you are saying mere section names extracted from file names where the listed variables come from must match supported versions. the sections are there for reader's convenience. when i did research on `ansible` and how people write roles, lots of them do not explain default values of variables. they simply say "see files for defaults", which is just annoying. I do not think I have promised the sections names in question are supported versions.

why are you trying to be so specific while I am trying to write generic roles that could work for non-supported versions? what is the reason? according to your suggestions, all the `README.md` should show specific supported versions. if "Ubuntu-14 means Ubuntu-14.04 and Ubuntu-14.10 which are the different releases", writing "RedHat" in `README.md` is wrong. neither "RedHat 7". it must have been "RedHat 7.3".